lib/partitioninfo.coffee

```diff
@@ -30,3 +33,40 @@ exports.get = (image, definition) ->
 		return Promise.props
 			offset: partition.getPartitionOffset(parsedPartition)
 			size: partition.getPartitionSize(parsedPartition)
+
+###
+# @summary Read all partition tables from a disk image recursively.
+# @public
+# @function
+#
+# @param {String} image - image path
+# @param {Integer} offset - where the first partition table will be read from, in bytes
+#
+# @returns {Promise<Array<Object>>} partitions information
+#
+# @example
+# partitioninfo.getPartitions 'foo/bar.img',
+# .then (information) ->
+# 	for partition in information
+# 		console.log(partition.offset)
+# 		console.log(partition.size)
+###
+exports.getPartitions = getPartitions = (image, offset = 0) ->
+	Promise.try ->
+		if not offset
```

Comment:
I believe the intention here is for this condition to return true if the offset is zero right? Maybe worth to check against `0` to make it more obvious, and avoid any confusion later on?